include/gpb.hrl

```diff
@@ -1 +0,0 @@
-../_build/default/plugins/gpb/include/gpb.hrl
```

Comment:
👀

Reply:
Yeah… it was not needed anymore.
The source one will be removed in [RTI-14088](https://adroll.atlassian.net/browse/RTI-14088).